specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/examples/Events_ListBySubscriptionId.json

```diff
@@ -0,0 +1,110 @@
+              },
+              "links": [
+                {
+                  "type": "Hyperlink",
+                  "displayText": {
+                    "value": "Request RCA",
+                    "localizedValue": "Request RCA"
+                  },
+                  "extensionName": "Microsoft_Azure_Health",
+                  "bladeName": "RequestRCABlade",
+                  "parameters": {
+                    "trackingId": "BC_1-FXZ",
+                    "rcaRequested": "False"
+                  }
+                },
+                {
+                  "type": "Button",
+                  "displayText": {
+                    "value": "Sign up for updates",
+                    "localizedValue": "Sign up for updates"
+                  },
+                  "extensionName": "Microsoft_Azure_Health",
+                  "bladeName": "AzureHealthBrowseBlade",
+                  "parameters": {
+                    "trackingId": "BC_1-FXZ"
+                  }
+                }
+              ],
+              "level": "Warning",
+              "eventLevel": "Informational",
+              "impactStartTime": "2018-11-07T00:00:00Z",
+              "impactMitigationTime": "2018-11-08T00:00:00Z",
+              "impact": [
+                {
+                  "impactedService": "Virtual Machines",
+                  "impactedRegions": [
+                    {
+                      "impactedRegion": "West US",
+                      "status": "Active",
+                      "impactedSubscriptions": [
+                        "{subscriptionId}"
+                      ],
+                      "impactedTenants": [],
+                      "lastUpdateTime": "2017-12-05T21:05:00Z"
+                    }
+                  ]
+                }
+              ],
+              "recommendedActions": {
+                "message": "Recommended actions title",
+                "actions": [
+                  {
+                    "groupId": 23243,
+                    "actionText": "action 1"
+                  },
+                  {
+                    "groupId": 23432,
+                    "actionText": "action 2"
+                  }
+                ],
+                "localeCode": "en"
+              },
+              "faqs": [
+                {
+                  "question": "This is a question",
+                  "answer": "This is an answer",
+                  "localeCode": "en"
+                }
+              ],
+              "isHIR": false,
+              "enableMicrosoftSupport": true,
+              "enableChatWithUs": false,
+              "priority": 2,
+              "lastUpdateTime": "2018-11-08T00:00:00Z",
+              "hirStage": "resolved",
+              "maintenanceId": "{maintenanceId}",
+              "maintenanceType": "{maintenanceType}",
+              "argQuery": "{argQuery}"
```

Comment:
are these realistic enough values that you would want to show in your example docs?

Reply:
These are not realistic values, will try to add sample values for the examples. Thank you for your suggestion.